docs/release-notes/8.4.5-5.md

```diff
@@ -0,0 +1,116 @@
+
+**Changes default values of MyRocks variables**
+
+* [`--rocksdb_disable_instant_ddl`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_disable_instant_ddl) - the default value is changed from `ON` to `OFF`.
+
+* [`--rocksdb_file_checksums`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_file_checksums) - the data type is changed from `Boolean` to `ENUM`. Also, the default value is changed from `OFF` to `CHECKSUMS_OFF`.
+
+* [`--rocksdb_compaction_readahead_size`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_compaction_readahead_size) - the default value is changed from `0` (zero) to `2097152`.
+
+**Deprecates MyRocks variable**
+
+* [`--rocksdb_disable_instant_ddl`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_disable_instant_ddl) - this variable is being deprecated and is expected to be removed in a future release.
+
+**Removes MyRocks variables**
+
+* [`--rocksdb-access-hint-on-compaction-start`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_access_hint_on_compaction_start)
+* [`--rocksdb_large_prefix`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_large_prefix)
+* [`--rocksdb_strict_collation_check`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_strict_collation_check)
+* [`--rocksdb_strict_collation_exceptions`](https://docs.percona.com/percona-server/8.4/myrocks-server-variables.html#rocksdb_strict_collation_exceptions)
+
+### MySQL 8.4.5
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.4.5 and included in Percona Server for MySQL are the following:
+
+* Fixed an issue where `CHECK TABLE` sometimes incorrectly reported that spatial indexes were corrupted. (Bug #37286473)
+
+* Fixed an issue in InnoDB redo log recovery to improve data safety after a crash. (Bug #37061960)
+
+* Fixed an issue where reading `index_id` values could lead to incorrect behavior with indexes. (Bug #36993445, Bug #37709706)
+
+* Fixed a bug related to the `lower_case_table_names` setting that caused inconsistent behavior with table names on different systems. (Bug #32288105)
+
+* Fixed a bug where `mysqldump` did not properly escape certain special characters in its output. (Bug #37540722, Bug #37709163)
+
+* The `fprintf_string()` function in `mysqldump` did not use the correct quote character for escaping strings. (Bug #37607195)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.4.5 release notes](https://dev.mysql.com/doc/relnotes/mysql/8.4/en/news-8-4-5.html).
+
+## Improvements
+
+[PXC-4645](https://perconadev.atlassian.net/browse/PXC-4645): Improves State Snapshot Transfer (SST) failure diagnostics. `garbd` now uses distinct exit codes to differentiate between donor exit, SST script failure, and garbd-initiated termination, making SST issues easier to identify and debug.
```

Comment:
This change was for our internal purposes (PXC Operator). I'm not sure if we included it in 8.0.42 RN. If yes - let's keep it here, if not - let's remove.